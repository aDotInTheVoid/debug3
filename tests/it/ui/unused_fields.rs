#![deny(dead_code)] // Becuase ui-test requires an error, not warning

// Don't change line nos
pub struct Foo {
    x: i32,
    y: i32,
}

pub fn foo_x(foo: &Foo) -> i32 {
    foo.x
}

fn main() {}
