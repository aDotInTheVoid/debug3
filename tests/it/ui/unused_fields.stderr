error: field `y` is never read
 --> tests/it/ui/unused_fields.rs:6:5
  |
4 | pub struct Foo {
  |            --- field in this struct
5 |     x: i32,
6 |     y: i32,
  |     ^
  |
note: the lint level is defined here
 --> tests/it/ui/unused_fields.rs:1:9
  |
1 | #![deny(dead_code)] // Becuase ui-test requires an error, not warning
  |         ^^^^^^^^^
